src/components/FieldLabel/index.jsx

```diff
@@ -10,21 +11,26 @@ const FieldLabel = (props) => {
   }
 
   return (
-    <div className={props.className}>
-      <strong>{props.text}:</strong>
+    <div id={props.id} className={props.className}>
+      <strong>{props.text}</strong>
+      {props.helpText && <FieldHelp id={`${props.id}-help`} tip={props.helpText} />}
       {requireText}
     </div>
   );
 };
 
 FieldLabel.defaultProps = {
   className: '',
+  helpText: '',
+  id: 'label', // will not be suitable for production, but useful default for tests
```

Comment:
Why not just make it a required field?

Reply:
Because I was being _laaaaazy_. I can fix.

Reply:
Oh actually - because we haven't converted all the field labels to have helpText yet - and the id is only required if you pass helpText.

So maybe I'll just set the default to null and only show helpText if it's set.